nle/nethack/nethack.py

```diff
@@ -29,6 +29,8 @@
     "program_state": dict(shape=PROGRAM_STATE_SHAPE, dtype=np.int32),
     "internal": dict(shape=INTERNAL_SHAPE, dtype=np.int32),
     "inv_glyphs": dict(shape=INV_GLYPHS_SHAPE, dtype=np.int16),
+    "inv_letters": dict(shape=INV_GLYPHS_SHAPE, dtype=np.uint8),
+    "inv_oclasses": dict(shape=INV_GLYPHS_SHAPE, dtype=np.uint8),
```

Comment:
Copypaste typo for shape?

Reply:
Good point. It's literally the same shape but we may want to make that more or less obvious.

Once we have the strings as well their shape will be different.

Reply:
(Fixed in https://github.com/facebookresearch/nle/pull/73).